web/libs/datamanager/src/components/DataManager/Toolbar/instruments.jsx

```diff
@@ -70,20 +70,23 @@ export const instruments = {
     return <ViewToggle size={size} style={style} />;
   },
   columns: ({ size }) => {
-    const iconProps = {};
-    const isNewUI = isFF(FF_LOPS_E_10);
-
-    if (isNewUI) {
+    const iconProps = {
```

Comment:
short term this is fine - but ideally I think this kind of stuff needs to get into the css and we toggle on css classes - maybe part of a greater initiative as I see this alot through out our code